io_scene_dae/__init__.py

```diff
@@ -23,7 +23,7 @@
     "name": "OpenMW Collada Exporter",
     "author": "OpenMW community, Godot Engine team",
     "version": (1, 10, 11),
-    "blender": (2, 83, 0),
+    "blender": (2, 93, 0),
```

Comment:
Why the version bump?

Reply:
My mistake, thought the issue with the changed python API came in 2.93, but it was actually back with 2.83, so that's the min required version.